src/Sulu/Bundle/AdminBundle/Resources/js/containers/Toolbar/Toolbar.js

```diff
@@ -140,23 +144,23 @@ class Toolbar extends React.Component<ToolbarProps> {
                                 success={!onNavigationButtonClick && showSuccess}
                             />
                         }
-                        {this.toolbarStore.hasItemsConfig() &&
+                        {!!itemsConfig && itemsConfig.length > 0 &&
                             <ToolbarComponent.Items>
                                 {this.itemsConfig.map((itemConfig, index) => getItemComponentByType(itemConfig, index))}
                             </ToolbarComponent.Items>
                         }
                     </ToolbarComponent.Controls>
                     <ToolbarComponent.Controls>
-                        {this.toolbarStore.hasIconsConfig() &&
+                        {!!iconsConfig &&
```

Comment:
missing `iconsConfig.length` check and think `!!iconsConfig` is not needed as always an empty array is now returned which is always be true?